client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html

```diff
@@ -0,0 +1,47 @@
+<div class="oui-modal oui-modal_shadow">
+    <div class="oui-modal__header">
+        <button
+            class="oui-icon oui-icon-close_thin oui-modal__close-button"
+            type="button"
+            aria-label="Exit"
+            ng-click="ctrl.cancel()"></button>
+    </div>
+    <div class="oui-modal__body oui-modal__body_no-icon">
+        <div class="oui-modal__body-text mb-5">
+            <h2 class="oui-modal__title" data-translate="logs_role_overview_title"></h2>
+        </div>
+        <dl class="preview-list preview-list_no-margin">
+            <dt class="preview-list__term"
+                data-translate="logs_role_overview_role"></dt>
+            <dd class="preview-list__definition"
+                data-ng-bind=":: ctrl.role.info.name"></dd>
+            <dt class="preview-list__term"
+                data-translate="logs_role_overview_description"></dt>
+            <dd class="preview-list__definition"
+                data-ng-bind=":: ctrl.role.info.description"></dd>
+            <dt class="preview-list__term"
+                data-translate="logs_role_overview_permissions"></dt>
+            <dd class="preview-list__definition">
+                <span data-ng-if="ctrl.role.permissions.length === 0"> - </span>
+                <div data-ng-if="ctrl.role.permissions.length > 0">
+                    <div data-ng-repeat="permission in ctrl.role.permissions">
```

Comment:
You should always have a "track by" with a ng-repeat, even if it's $index.

Reply:
@Netnix  the logic changed a bit. Have a look now. Have added track by index too